gitgud/__main__.py

```diff
@@ -317,21 +316,21 @@ def handle_init(self, args):
                 passargs = ''
 
             with open(path, 'w+') as hook_file:
-                hook_file.write('#!/bin/bash' + os.linesep)
+                hook_file.write('#!/bin/bash' + '\n')
```

Comment:
By what the [docs](https://docs.python.org/3/library/os.html#os.linesep) say, we shouldn't use os.linesep for this purpose.